Examples/Makefile.in

```diff
@@ -700,8 +700,10 @@ endif
 # Creating and building Javascript wrappers
 # ----------------------------------------------------------------
 
+NAPI_DIR=$(shell npm config get prefix)/lib/node_modules/node-addon-api
```

Comment:
The variables for running the examples are set at configure time, not runtime. Can you move this to configure.ac as part of the enabling of the napi ENGINE.